diff --git a/scripts/python3/compilation/compil_tools.py b/scripts/python3/compilation/compil_tools.py
index 22727fc20..b5b28e754 100644
--- a/scripts/python3/compilation/compil_tools.py
+++ b/scripts/python3/compilation/compil_tools.py
@@ -1012,7 +1012,7 @@ def compile_api_f2py(name, api_dir, source_list, skip_source, \
                 'downgrade setuptools to version 73.0.'
             )

-        environ["FC"] = compiler
+        environ["FC"] = fcompiler
         cmd = f'{f2py_name} {quiet} -c {pyf_file} --backend meson '\
               f'{compile_cmd} -I{path.join(api_dir, "include")}'\
               f'{ld_flags} --build-dir {f2py_build_dir}'
